hvpy/core.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+def parse_response(response: requests.Response, output_parameters: OutputType):
+
+    if output_parameters == OutputType.Raw:
+        return response.content
+    elif output_parameters == OutputType.String:
+        return str(response.content)
+    elif output_parameters == OutputType.Json:
+        return response.json()
+
+
+def execute_api_call(url: str, input_parameters: HvpyParameters):
+
+    response = requests.get(url, params=input_parameters.dict())
```

Comment:
Once the url is moved into getJP2ImageInputParameters, you can do:
```suggestion
    response = requests.get(input_parameters.url, params=input_parameters.dict())
```

Reply:
`Requests` throws an error passing `input_parameters.url` directly as `url=`.

Reply:
Yes, because you made `input_paramters.url` a function. Initially I was thinking it would just be a field in the class.
This is good though.